src/test/java/com/microsoft/aad/msal4j/OAuthRequestValidationTest.java

```diff
@@ -222,10 +228,10 @@ public void oAuthRequest_for_acquireTokenByAsymmetricKeyCredential() throws Exce
         Assert.assertFalse(StringUtils.isEmpty(queryParams.get("client_assertion")));
 
         // to do validate scopes
-        Assert.assertEquals(SCOPES, queryParams.get("scope"));
-
-        Assert.assertEquals(CLIENT_ASSERTION_TYPE_JWT, queryParams.get("client_assertion_type"));
-        Assert.assertEquals(ON_BEHALF_OF_USE_JWT, queryParams.get("requested_token_use"));
+        //TODO for OBO tests, should query params include default scopes?
+        Assert.assertEquals( queryParams.get("scope"), SCOPES);
```

Comment:
This test is currently failing - should OBO requests include default scopes (openid profile offline_access)? Every other request has them.

Reply:
yes should @sangonzal

Reply:
@jmprieur thanks for confirming. I have updated the unit test to reflect this.